label_errors.py

```diff
@@ -0,0 +1,184 @@
+
+    for job_name, job in jobs.items():
+        content_input = job.get("content", {}).get("input")
+        ml_task = job.get("mlTask")
+        if (
+            content_input == ContentInput.Radio
+            and input_type == InputType.Image
+            and ml_task == MLTask.Classification
+        ):
+            job_path = build_model_repository_path(HOME, project_id, job_name, "")
+            data_path = os.path.join(job_path, "data")
+            model_path = os.path.join(job_path, "model")
+
+            if clear_dataset_cache and os.path.exists(job_path) and os.path.isdir(job_path):
+                kili_print("Dataset cache for this project is being cleared.")
+                shutil.rmtree(job_path)
+
+            os.makedirs(job_path, exist_ok=True)
+            os.makedirs(data_path, exist_ok=True)
+            os.makedirs(model_path, exist_ok=True)
+
+            kili_print("Downloading datasets from Kili")
+            assets = get_assets(kili, project_id, parse_label_types(label_types), max_assets)
+            if len(assets) == 0:
+                raise Exception("No asset in dataset, exiting...")
+
+            for asset in tqdm(assets):
+                tic = time.time()
+                n_try = 0
+                while n_try < 20:
+                    try:
+                        img_data = requests.get(
+                            asset["content"],
+                            headers={
+                                "Authorization": f"X-API-Key: {api_key}",
+                            },
+                        ).content
+                        break
+                    except Exception:
+                        time.sleep(1)
+                        n_try += 1
+                img_path = os.path.join(
+                    data_path, asset["labels"][0]["jsonResponse"][job_name]["categories"][0]["name"]
+                )
+                os.makedirs(img_path, exist_ok=True)
+                with open(os.path.join(img_path, asset["id"] + ".jpg"), "wb") as handler:
+                    handler.write(img_data)
+                toc = time.time() - tic
+                throttling_per_call = 60.0 / 250  # Kili API calls are limited to 250 per minute
+                if toc < throttling_per_call:
+                    time.sleep(throttling_per_call - toc)
+
+            model_name = set_default(
+                model_name,
+                ModelName.EfficientNetB0,
+                "model_name",
+                [ModelName.EfficientNetB0, ModelName.Resnet50],
+            )
+
+            found_errors = train_and_get_error_labels(
+                cv_n_folds=cv_folds,
+                data_dir=data_path,
+                model_dir=model_path,
+                model_name=model_name,
+                training_epochs=training_epochs,
+            )
+
+            print()
+            kili_print("Number of wrong labels found: ", len(found_errors))
+
+            if found_errors:
```

Comment:
add the code below into a `upload_errors` method in `utils.helpers`

Reply:
Added two methods, one for saving the json and one for actually uploading to Kili